script.js

```diff
@@ -22,4 +22,145 @@ closeAbout.addEventListener('click', () => {
+    tec: ['html', 'css', 'javaScript'],
+    image: './Images/Works/SnapshootPortfolio1.png',
+    link: 'https://www.github.com',
+    linkLive: 'https://alejandrakeber.github.io/',
+    linkUrl: 'https://github.com/AlejandraKeber/Project-setup-and-mobile-main',
+  },
+  {
+    title: 'Mutli-Post Stories',
+    details: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    popupdescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    tec: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    image: './Images/Works/SnapshootPortfolio2.png',
+    link: 'https://www.github.com',
+    linkLive: 'https://alejandrakeber.github.io/',
+    linkUrl: 'https://github.com/AlejandraKeber/Project-setup-and-mobile-main',
+  },
+  {
+    title: 'Facebook 360',
+    details: ['CANOPY', 'Full Stack Dev', '2015'],
+    description: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    popupdescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    tec: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    image: './Images/Works/SnapshootPortfolio3.png',
+    link: 'https://www.github.com',
+    linkLive: 'https://alejandrakeber.github.io/',
+    linkUrl: 'https://github.com/AlejandraKeber/Project-setup-and-mobile-main',
+  },
+  {
+    title: 'Uber Navigation',
+    details: ['Uber', 'Lead Developer', '2018'],
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    popupdescription: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    tec: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    image: './Images/Works/SnapshootPortfolio4.png',
+    link: 'https://www.github.com',
+    linkLive: 'https://alejandrakeber.github.io/',
+    linkUrl: 'https://github.com/AlejandraKeber/Project-setup-and-mobile-main',
+  },
+];
+
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure your design sticks as much as possible to the form of the Figma template(e.g.:** background color, **[the blur](https://www.w3schools.com/howto/howto_css_blurred_background.asp)** on the background, Styling of the "See Live" and "See Source" buttons on the desktop version,...**)
#### Fogma Desktop Design
![Screenshot from 2022-09-09 01-34-12](https://user-images.githubusercontent.com/83042742/189243829-29dde3bf-28cc-417d-91a3-0dae91107001.png)

#### Your Desktop Design
![Screenshot from 2022-09-09 01-34-51](https://user-images.githubusercontent.com/83042742/189244343-d0c80724-f80a-44c8-ad69-4a0170a0a4e2.png)

| Figma Mobile Design | Your Mobile Design |
| ----------------------------- | ---------------------------- |
| ![Screenshot from 2022-09-09 01-43-22](https://user-images.githubusercontent.com/83042742/189244433-8acaa26b-0d37-47c8-8d27-435e5ab845db.png) | ![Screenshot from 2022-09-09 01-47-38](https://user-images.githubusercontent.com/83042742/189244797-0d034b39-fdd6-4b82-8f2c-e1bed38c5c4a.png) |